modules/docs/mcp/config.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Configuration for MCP resources
+ * Single source of truth for all MCP resource file names
+ */
+
+export const bestPracticesResourcesFiles = [
+  'COMPOUND_COMPONENTS.mdx',
+  'CREATING_COMPOUND_COMPONENTS.mdx',
```

Comment:
ooo i see potential for examples/accessibility docs here too as well as styling best practices